src/ir/possible-contents.cpp

```diff
@@ -1359,8 +1359,13 @@ struct Flower {
   // exists it is not added.
   void connectDuringFlow(Location from, Location to);
 
-  // Contents sent to a global location can be filtered in a special way during
-  // the flow, which is handled in this helper.
+  // Contents sent to certain locations can be filtered in a special way during
+  // the flow, which is handled in these helpers. These may update
+  // |worthSendingMore| which is whether it is worth sending any more content to
+  // this location in the future.
+  void filterExpressionContents(PossibleContents& contents,
+                                const ExpressionLocation& exprLoc,
+                                bool& worthSendingMore);
```

Comment:
Why not return the bool?

Reply:
Keeping it as an out param keeps the name. And the return value would not be related to the function's name.

Reply:
Oh I see, `worthSendingMore` is some global-ish state that all these helper functions essentially close over. I would probably still prefer a purer approach, but this makes sense.